.github/workflows/ci-cd.yml

```
@@ -0,0 +1,147 @@
+    if: github.ref == 'refs/heads/develop'
+    environment: development
+    
+    steps:
+    - name: 코드 체크아웃
+      uses: actions/checkout@v4
+      
+    - name: 개발 환경 배포 설정
+      run: |
+        echo "SPRING_BOOT_BASE_URL=${{ secrets.DEV_SPRING_BACKEND_URL }}" >> .env.development
+        echo "REDIS_URL=${{ secrets.DEV_REDIS_URL }}" >> .env.development
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)